interface/forms/care_plan/new.php

```diff
@@ -146,99 +163,111 @@ function set_related(codetype, code, selector, codedesc) {
+							</div>
+							<div class="clearfix"></div>
+						</div>
+						
+					</div>
+					<?php
+						}
+					} else {
+					?>
+					<div class="tb_row" id="tb_row_1">
+						<div class="form-group">
+							<div class=" forms col-xs-3">
+								<label for="code_1" class="h5"><?php echo xlt('Code'); ?>:</label>
+								<input type="text" id="code_<?php echo attr($key) + 1; ?>"  name="code[]" class="form-control code" value="<?php echo text($obj{"code"}); ?>"  onclick='sel_code(this.parentElement.parentElement.id);'>
+								<span id="displaytext_1"  class="displaytext help-block"></span>
+								<input type="hidden" id="codetext_1" name="codetext[]" class="codetext" value="<?php echo text($obj{"codetext"}); ?>">
+							</div>
+							<div class="forms col-xs-4">
+								<label for="description_1" class="h5"><?php echo xlt('Description'); ?>:</label>
+								<textarea name="description[]"	id="description_1" class="form-control description"  rows="3" ><?php echo text($obj{"description"}); ?></textarea>
+							</div>
+							<div class="forms col-xs-2">
+								<label for="code_date_1" class="h5"><?php echo xlt('Date'); ?>:</label>
+								<input type='text' id="code_date_1"  name='code_date[]' class="form-control code_date datepicker" <?php echo attr($disabled) ?> value='<?php echo attr($obj{"date"}); ?>' title='<?php echo xla('yyyy-mm-dd Date of service'); ?>' />
+							</div>
+							<div class="forms col-xs-2">
+								<label for="care_plan_type_1" class="h5"><?php echo xlt('Type'); ?>:</label>
+								<select name="care_plan_type[]" id="care_plan_type_1" class="form-control care_plan_type">
+									<option value=""></option>
+									<?php foreach($care_plan_type as $value):
+									  $selected = ($value['value'] == $obj{"care_plan_type"}) ? 'selected="selected"' : '';
+									?>
+									<option value="<?php echo attr($value['value']);?>" <?php echo $selected;?>><?php echo text($value['title']);?></option>
+									<?php endforeach;?>
+								</select>
+							</div>
+							<div class="forms col-xs-1 " style="padding-top:35px">
+								<i class="fa fa-plus-circle fa-2x" aria-hidden="true" onclick="duplicateRow(this.parentElement.parentElement.parentElement);" title='<?php echo xla('Click here to duplicate the row'); ?>'></i>
+								<i class="fa fa-times-circle fa-2x text-danger"  aria-hidden="true" onclick="deleteRow(this.parentElement.parentElement.parentElement.id);"  title='<?php echo xla('Click here to delete the row'); ?>'></i>
+							</div>
+							<div class="clearfix"></div>
+						</div>
+					</div>
+						
+					<?php }
+					?>
+				
+				</fieldset>
+				<div class="form-group">
+					<div class="col-sm-12 text-center">
+						<div class="btn-group" role="group">
+							<a href="javascript:top.restoreSession();document.my_form.submit();" class="btn btn-default btn-save"><?php echo xlt('Save'); ?></a>
```

Comment:
I thought we weren't converting a to button?
(note that when I did convert some a tags to buttons, things did break; the behavior does not seem to be the same)